src/components/summary.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { IStateOptions } from "../constants/types";
+import { attributeOptions } from "../constants/constants";
+
+interface IProps {
+  category: string;
+  selectedOptions: IStateOptions;
+}
+
+export const Summary: React.FC<IProps> = ({category, selectedOptions}) => {
+
+  const getSummaryText = () => {
+    if (category === "Place") {
+      const place = selectedOptions.geographicLevel || "";
+      const states = selectedOptions.states.join(`, `);
+      const colon = place && states ? ": " : "";
+      return (
+        `${place}${colon}${states}`
+      );
+    } else if (category === "Attributes") {
+        const resultString = attributeOptions.filter((attr) => {
+          const value = selectedOptions[attr.key];
+          const valueIsArrayWithLength = Array.isArray(value) && value.length > 0;
+          const valueIsDefined = typeof value === "string" && value;
+          return valueIsArrayWithLength || valueIsDefined;
+        }).map((attr) => {
+          const value = selectedOptions[attr.key];
+          const valueIsArrayWithLength = Array.isArray(value) && value.length > 0;
+          const valueIsDefined = typeof value === "string" && value;
+          const label = attr.label && (valueIsArrayWithLength || valueIsDefined) ? `${attr.label}: ` : "";
+
+          if (Array.isArray(value) && value.length > 0) {
```

Comment:
You can also use `valueIsArrayWithLength`
```suggestion
          if (valueIsArrayWithLength) {
```